roles/master-image/tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+
+- name: Download the source from GitHub
+  git:
+    repo: https://github.com/ocurrent/macos-infra
+    dest: "{{ ansible_env.HOME }}/macos-infra"
+    force: yes
+
+- name: chmod +x
+  file:
+    dest: "{{ ansible_env.HOME }}/macos-infra/scripts/{{ item }}"
+    mode: a+x
+  loop:
+    - new-user.sh
+    - macos-homebrew-ocaml.sh
+    - cleanup.sh
+
+- name: Create a scoreboard directory if it does not exist
+  file:
+    path: "{{ ansible_env.HOME }}/scoreboard"
+    state: directory
+    mode: '0755'
+
+- name: Check for user mac1000
+  shell: dscl . read /Users/mac1000
```

Comment:
It could be but the prefix `mac` is hard coded in `obuilder/lib/sandbox.macos.ml'

  let user = user_name ~prefix:"mac" ~uid:t.uid in

I have updated the playbook to use `{{ user_name }}` in place of `mac1000`